src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Guid.txt

```diff
@@ -0,0 +1,79 @@
+﻿// Guid function 
+// https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/functions/function-guid
+
+// Guid with no args is volatile, so we can't directly test the output
+
+>> IsBlank(GUID())
+false
+
+>> IsError(GUID())
+false
+
+// Multiple invocations are unique
+>> GUID() <> GUID()
+true
+
+// Coercion to string is 32 chars + 4 hyphens
+>> Len(GUID())
+36
+
+>> GUID(Blank())
+Blank()
+
+>> GUID(Text(1/0))
+#Error(Kind=Div0)
```

Comment:
Have we decided on a format for how to specify kind and count for errors?

Reply:
@CarlosFigueiraMSFT did a little while back, yes.